src/lib/components/Header/index.svelte

```diff
@@ -1,16 +1,24 @@
 <script lang="ts">
 	import { page } from '$app/stores';
+	import { goto } from '$app/navigation';
 	import { createEventDispatcher } from 'svelte';
 	const dispatch = createEventDispatcher();
 	// TODO Connect to wallet state.
 	let provider = true;
 </script>
 
 <header>
-	<div class:active={$page.url.pathname === '/'}><a sveltekit:prefetch href="/">Home</a></div>
+	<div class="home" class:active={$page.url.pathname === '/'}>
+		<a sveltekit:prefetch href="/"><h3>👔 Workstreams</h3></a>
+		<button on:click={() => dispatch('createAction')}>Create</button>
+	</div>
 
-	<div class="corner">
-		<button on:click={() => dispatch('createAction')}>Create workstream</button>
+	<div class="user">
+		<a
+			href="/my-workstreams"
```

Comment:
Do we want to allow other users to see the workstreams of someone else?
If so we should use a dynamic route, using some identifier.

I.e. `/<userAddress>` instead of `/my-workstreams`.

Reply:
That's a good point. I think that would be cool.

Reply:
We should probably do that in a follow up PR once we have the wallet address, no? The url could be something like `/address/<ethAddress>`

Reply:
Agree.

Reply:
created an issue: https://github.com/radicle-dev/workstreams-app/issues/9